components/eam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -190,6 +190,10 @@ subroutine micro_p3_readnl(nlfile)
   select case (trim(micro_p3_tableversion))
     case ('4')
        ! Version 4 is valid
+    case ('4.1')
+      ! Version 4.1 is valid
+    case ('4.1.1')
+      ! Version 4.1.1 is valid  
```

Comment:
Do we really want to continue supporting v4 and v4.1 if we think they are wrong? I'd advocate for only supporting v4.1.1.